framework/src/modules/http_api/api_error.js

```diff
@@ -24,23 +24,24 @@
  * @see Parent: {@link helpers}
  * @todo Add description for the params
  */
-function ApiError(message, code, errors = []) {
-	this.message = message;
-	this.code = code;
-	this.errors = errors;
-}
-
-ApiError.prototype = new Error();
+class ApiError extends Error {
+	constructor(message, code, errors = []) {
+		super();
+		this.message = message;
+		this.code = code;
+		this.errors = errors;
+	}
 
-/**
- * Description of the function.
- *
- * @todo Add @returns tag
- */
-ApiError.prototype.toJson = function() {
-	return {
-		message: this.message,
-	};
-};
+	/**
+	 * Returns an ApiError instance message
+	 *
+	 * @returns {string}
+	 */
+	toJson() {
```

Comment:
Prefer `toJSON`

Reply:
@michielmulders this are all old methods so I'm just removing `prototype` usage and we'll refactor this classes more in the coming weeks. Please create an issue for this comment.